content/blog/welcome-erin-2021-momentum.md

```diff
@@ -8,6 +8,7 @@ categories:
 tags:
   - impact
   - newsletter
+  - noaa-fisheries
```

Comment:
I think this one we don't need to tag with `noaa-fisheries`. But I am new enough to github reviews that I don't know how to reject this, I will try.